src/context/WalletContext.jsx

```diff
@@ -1,9 +1,21 @@
 import React, { createContext, useState, useContext } from 'react'
 
+import { getSession } from '../session'
+
 const WalletContext = createContext()
 
+const initialWalletFromSession = () => {
+  const session = getSession()
+  return !(session && session.name && session.token)
+    ? null
+    : {
+        name: session.name,
+        token: session.token,
+      }
+}
```

Comment:
```suggestion
  return session && session.name && session.token
    ? {
        name: session.name,
        token: session.token,
      }
    : null
```

Reply:
I try to keep the longer expressions at the end, think it is easier for the eyes. But I get your point. Applied!